<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreGame</class>
 <widget class="QWidget" name="PreGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreGame</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="boardTab">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;選擇盤面配置，可以選擇標準盤面或輸入FEN碼&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <attribute name="title">
       <string>盤面</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="regular_radio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;標準的西洋棋盤面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>標準盤面</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QRadioButton" name="FEN_radio">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;使用FEN碼來載入盤面，分便覆盤&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>FEN碼：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FEN_edit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FEN格式： &amp;lt;棋子擺放&amp;gt; &amp;lt;移動方&amp;gt; &amp;lt;可否入堡&amp;gt; &amp;lt;過路兵的目標&amp;gt; &amp;lt;halfmove clock&amp;gt; &amp;lt;fullmove number&amp;gt;&lt;/p&gt;&lt;p&gt;每一個項目中間以一個空格分隔&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timeTab">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;設定時間限制&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <attribute name="title">
       <string>時間</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="noTimeLimit_radio">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;沒有時間限制，無拘無束的進行遊戲&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>沒有時間限制</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hasTimeLimit_radio">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;每人有固定的思考時間，用完即自動投降&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>有時間限制（每人）</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>小時</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="hour">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>分鐘</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minute">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>秒</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="second">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startGame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>FEN_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>FEN_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noTimeLimit_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>hour</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noTimeLimit_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>minute</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noTimeLimit_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>second</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
